src/PooledArrays.jl

```diff
@@ -474,7 +448,8 @@ Base.@propagate_inbounds function Base.getindex(A::PooledArray, I::Union{Real, A
     return PooledArray(RefArray(newrefs), A.invpool, A.pool, A.refcount)
 end
 
-Base.@propagate_inbounds function Base.getindex(A::SubArray{T,N,<:PooledArray}, I::Union{Real, AbstractVector}...)
+Base.@propagate_inbounds function Base.getindex(A::SubArray{<:Any,<:Any,<:PooledArray},
```

Comment:
Likewise, here, can't you just use `DataAPI.refarray` to write a single method common with `PooledArray`?

Reply:
I will check. I was getting dispatch ambiguities here, but maybe this will work. It is always hard to extend methods from Julia Base.

Reply:
I was getting dispatch ambiguities.